app/controllers/api/v2/providers_controller.rb

```diff
@@ -103,6 +104,24 @@ def get_user
         @user = User.find(params[:user_id])
       end
 
+      def update_accrediting_enrichment
+        return unless accredited_bodies_params.values.any?
```

Comment:
Ruby also provides a `none?` that might be better here:

```suggestion
        return if accredited_bodies_params.values.none?
```

Reply:
TIL 😲

Reply:
Checking what versions of ruby has it:

```
|       | `Enumerable#none?` | `Array#none?` |
| 2.5.x |         ✅         |       ❌      |
| 2.6.x |         ✅         |      ✅       | 
```

Reply:
tests passes, @misaka  vindicated... should really check 1.8 it was so much less fluffy...